### PR TITLE
Moneropedia: initial Kovri-related entries

### PR DESCRIPTION
**Pull request by anonimal**
I believe this should fulfill the base-line Moneropedia requirement for https://forum.getmonero.org/9/work-in-progress/2584/create-end-user-kovri-documentation.

I plan on adding at least a few more entries (and improving others) in the near future but wanted to get this merged a.s.a.p for community review.

To reiterate: these entries were intended for end-users, new developers, and are *not* specifications nor developer/API documentation in **any** remote sense of those words. *That* kind of documentation will come later and are out of the scope of the initial proposal.

All review is greatly appreciated, entry requests are welcome, and all-around questions/comments welcome.

Referencing:
  - monero-project/kovri#256
  - monero-project/monero-site#155

### CODE REVIEW

File: knowledge-base/moneropedia/address-book.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+tags: ["kovri"]
+terms: ["Address-Book"]
+summary: "Allows you to visit I2P websites/services that have the .i2p domain"
+---
+
+### The Basics
+
+In order to browse @I2P sites or services with @Kovri, you'll need an address book. An address book will allow you to translate @I2P websites/services that use the `.i2p` [top-level domain](https://en.wikipedia.org/wiki/Top_level_domain) into an address that @I2P network will understand.
+
+Without an address book, you would be stuck using a @base32-address every time visited an @I2P website/service - and that's not fun!
```
Comment (expez) [inline knowledge-base/moneropedia/address-book.md]: > every time visited

Missing a word here

Comment (anonimal) [inline knowledge-base/moneropedia/address-book.md]: Thanks, @expez

**Comment by anonimal** [comment]
*[I propose a two milestones, 50 XMR for each milestone, for a total of 100 XMR. The first milestone being a completed first draft that I will keep in my fork. The second milestone being completed, uploaded, and verified content.](https://forum.getmonero.org/6/ideas/2584/create-end-user-kovri-documentation)*

@fluffypony when will I receive a payout for the 1st milestone?

**Comment by anonimal** [comment]
Funds received, thank you @fluffypony.

File: knowledge-base/moneropedia/address-book.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+In order to browse @I2P sites or services with @Kovri, you'll need an address book. An address book will allow you to translate @I2P websites/services that use the `.i2p` [top-level domain](https://en.wikipedia.org/wiki/Top_level_domain) into an address that @I2P network will understand.
+
+Without an address book, you would be stuck using a @base32-address every time you visit an @I2P website/service - and that's not fun!
+
+### In-depth information
+
+Since [DNS](https://en.wikipedia.org/wiki/DNS) does not exist on the @I2P network, @Kovri also does **not** use DNS or any sort of @canonically-unique-host resolution. Instead, Kovri pairs a @locally-unique-host to a @base64-address @destination in a @subscription. Once your address book is filled with a @subscription, you can resolve your favorite `.i2p` domain site into a usable @I2P destination.
+
+### Creating an address Book
```
Comment (expez) [inline knowledge-base/moneropedia/address-book.md]: Weird capitalization of `Book`

File: knowledge-base/moneropedia/address-book.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+By default, your installation will come with a default public @subscription called `hosts.txt` in your @data-directory. When @Kovri starts, it loads this subscription and fetches any other subscriptions you've specified. Once loaded, your address book will be appropriately filled. For details on how to manage subscriptions, see @subscription.
+
+### Updating the address book
+
+Currently, there are several ways to update your address book:
+
+1. Use a @jump-service to insert into address book
```
Comment (expez) [inline knowledge-base/moneropedia/address-book.md]: Should be "insert into **the** address book"?

File: knowledge-base/moneropedia/canonically-unique-host.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+summary: "A host that is canonically resolved to an address or set of addresses"
+---
+
+### The Basics
+
+A Canonically-unique host is a [FQDN](https://en.wikipedia.org/wiki/FQDN) that will canonically resolve to a designated address or set of addresses. Not to be confused with a @locally-unique-host.
+
+### In-depth information
+
+A Canonically-unique host is defined by remote authoritative sources; usually through [DNS](https://en.wikipedia.org/wiki/DNS). When resolving a peer's hostname, you will most likely use an external source for resolution unless have the following implemented:
```
Comment (expez) [inline knowledge-base/moneropedia/canonically-unique-host.md]: Missing word in phrase "unless have the following implemented"

File: knowledge-base/moneropedia/destination.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+layout: moneropedia
+entry: "Destination"
+tags: ["kovri"]
+terms: ["Destination", "Destinations"]
+summary: "A in-net address that serves as a final endpoint (either local or remote)"
+---
+
+### The Basics
+
+A @destination is the @I2P @in-net address of the final endpoint you are trying to connect to (example: an @I2P website, service, or Monero node). This can also include a *local destination* of which *other* peers need to connect to in order to make contact and communication (similar to how, in @clearnet, your IP address is given to a website when you connect so it knows *where* to send the information back to).
```
Comment (expez) [inline knowledge-base/moneropedia/destination.md]: > make contact and communication 

Should be *communicate*?

File: knowledge-base/moneropedia/address-book.md (hunk 2 above)
Comment (anonimal) [inline knowledge-base/moneropedia/address-book.md]: Fixed, thanks.

File: knowledge-base/moneropedia/address-book.md (hunk 3 above)
Comment (anonimal) [inline knowledge-base/moneropedia/address-book.md]: Fixed, thanks.

File: knowledge-base/moneropedia/canonically-unique-host.md (hunk 4 above)
Comment (anonimal) [inline knowledge-base/moneropedia/canonically-unique-host.md]: Fixed, thanks.

File: knowledge-base/moneropedia/destination.md (hunk 5 above)
Comment (anonimal) [inline knowledge-base/moneropedia/destination.md]: Fixed, thanks.

File: knowledge-base/moneropedia/address-book.md [hunk 6]
```diff
@@ -16,15 +16,15 @@ Without an address book, you would be stuck using a @base32-address every time y
...
 
 By default, your installation will come with a default public @subscription called `hosts.txt` in your @data-directory. When @Kovri starts, it loads this subscription and fetches any other subscriptions you've specified. Once loaded, your address book will be appropriately filled. For details on how to manage subscriptions, see @subscription.
 
-### Updating the address book
+### Updating the Address Book
 
 Currently, there are several ways to update your address book:
 
-1. Use a @jump-service to insert into address book
+1. Use a @jump-service to insert I2P addresses into you address book
```
Comment (expez) [inline knowledge-base/moneropedia/address-book.md]: Your? :P

**Comment by expez** [comment]
Good read @anonimal! :+1:

**Comment by anonimal** [comment]
Thanks for the thorough review, @expez. You've proven that either my keyboard has stuck keys or I am [ESL](https://en.wikipedia.org/wiki/ESL) :laughing:

**Pull request merged**